pyanaconda/ui/gui/spokes/lib/custom_storage_helpers.py

```diff
@@ -383,9 +383,13 @@ def refresh(self, mountpoint, device, rootName, snapshots=False):
 
         if rootName and "_" in rootName:
             rootName = rootName.replace("_", "__")
+        delete_all_text = (C_("GUI|Custom Partitioning|Confirm Delete Dialog",
+                               "Delete _all other file systems in the %s root as well.") +
+                           "\n" +
+                           C_("GUI|Custom Partitioning|Confirm Delete Dialog",
+                               "(This includes those used by other installed operating systems.)"))
```

Comment:
How about "This includes those shared with other installed operating systems."

Or, following more closely @vpodzime suggestion, "This includes those also used by other..."